ui/app/store/actions.js

```diff
@@ -385,22 +385,28 @@ export function resetAccount () {
 }
 
 export function removeAccount (address) {
-  return dispatch => {
+  return async dispatch => {
     dispatch(showLoadingIndication())
 
-    return new Promise((resolve, reject) => {
-      background.removeAccount(address, (err, account) => {
-        dispatch(hideLoadingIndication())
-        if (err) {
-          dispatch(displayWarning(err.message))
-          return reject(err)
-        }
-
-        log.info('Account removed: ' + account)
-        dispatch(showAccountsPage())
-        resolve()
+    try {
+      await new Promise((resolve, reject) => {
+        background.removeAccount(address, async (error, account) => {
```

Comment:
We can drop the `async` here, yeah?

```suggestion
        background.removeAccount(address, (error, account) => {
```

Reply:
Yes, good point, thanks